src/logging/LoggedComand.java

```diff
@@ -0,0 +1,32 @@
+package logging;
+
+import edu.wpi.first.wpilibj2.command.Command;
+
+public class LoggedComand extends Command {
+
+    public static boolean IS_LOG = false;
+
+    private String commandName = getClass().toString();
+    private String requirements = getRequirements().toString();
+
+    @Override
+    public void initialize() {
+        if (IS_LOG) {
+            System.out.println("[" + System.currentTimeMillis() + "] - (" + requirements + ") " + commandName);
```

Comment:
If you type `end` at the end, why dont you type `start` here?

Reply:
The ones that don't say end are always start, it isn't necessary to specify twice